File: 2017级-理科-大学计算机基础/CX-北航大学计算机基础(17级理科)上机开放题目/BUAAOJ1207.py
ddl = [0] * 15
n = int(input())
remain = 48
week = 15 - n
for i in range(0, n):
    ddl[i] = int(input())
    remain -= ddl[i]

if week * 10 < remain:
    print("gg")
else:
    ans = int((remain + week - 1)/week)
    print(ans)

File: 2017级-理科-大学计算机基础/CX-北航大学计算机基础(17级理科)上机开放题目/BUAAOJ1207.c
#include <stdio.h>
int n, remain, week, x;
int main()
{
    scanf("%d", &n);
    remain = 48, week = 15 - n;
    while (n--)
        scanf("%d", &x), remain -= x;
    if (week * 10 < remain)
        puts("gg");
    else
        printf("%d", (remain + week - 1) / week);
}